.github/workflows/release-go-functions.yaml

```diff
@@ -35,9 +35,16 @@ jobs:
       - name: Get the version
         id: get_version
         run: echo ::set-output name=TAG_NAME::${GITHUB_REF#refs/tags/}
+      - name: BUild images
```

Comment:
If we all agree that the CD pipeline will live  outside of the repo, the github workflows for releasing go and ts fn will not be needed.